### PR TITLE
Homework 2 not working

### PR DESCRIPTION
**Pull request by camzillajs101**
The data stream is not changing when I reload the page. It must be the JS that is wrong. Please take a look at this and tell me what I did wrong!

### CODE REVIEW

**Comment by camzillajs101** [comment]
Oops
@rickyc (above PR)

File: homework-2.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+			Password: <input type="password" id="pw"><br>
+		</form>
+		<script>
+		$('form').submit(function(event){
+			let fn = $('#fn').val();
+			let ln = $('#ln').val();
+			let un = $('#un').val();
+			let em = $('#em').val();
+			let pw = $('#pw').val();
+			let url = "http://data.sparkfun.com/streams/lzDKN7V2dzFwDWrMxa78.json"
```
Comment (rickyc) [inline homework-2.html]: This isn't the right URL.

File: homework-2.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+			$.post(url, function(result){
+				result.stream._posts._doc.fields[email] = em;
```
Comment (rickyc) [inline homework-2.html]: Once you fix the url, the data will make more sense.

File: homework-2.html [hunk 3]
```diff
@@ -20,7 +20,7 @@
 			let un = $('#un').val();
 			let em = $('#em').val();
 			let pw = $('#pw').val();
-			let url = "http://data.sparkfun.com/streams/lzDKN7V2dzFwDWrMxa78.json"
+			let url = "http://data.sparkfun.com/output/lzDKN7V2dzFwDWrMxa78.json"
```
Comment (rickyc) [inline homework-2.html]: 🎉. You finally found the URL!

**Comment by rickyc** [comment]
**Required**
* [ ] Does the code work?
* [X] Was a SparkFun dataset with the appropriate fields created?
* [X] Is there a HTML / CSS form which the user can input the data?
* [ ] If the user presses the submit button, can does the data get logged onto SparkFun?
* [ ] Is the user able to view all the entries that have been logged onto SparkFun?
* [ ] Is the data being displayed in a table?
* [X] Is the code indented properly?

**Comment by camzillajs101** [comment]
@rickyc Merge?

**Comment by rickyc** [comment]
Still stuff missing

**Pull request merged**